app/src/main/java/ru/chaichuk/hwapp/db/MoviesTypeConverters.kt

```diff
@@ -0,0 +1,39 @@
+package ru.chaichuk.hwapp.db
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import ru.chaichuk.hwapp.data.Actor
+import ru.chaichuk.hwapp.data.Genre
+
+class MoviesTypeConverters {
+
+    @TypeConverter
+    fun fromGenres(genres : List<Genre>) : String {
+        val gson = Gson()
+        val type = object : TypeToken<List<Genre>>() {}.type
+        return gson.toJson(genres, type)
+    }
+
+    @TypeConverter
+    fun toGenres(genresJson : String) : List<Genre> {
+        val gson = Gson()
+        val type = object : TypeToken<List<Genre>>() {}.type
+        return gson.fromJson(genresJson, type)
+    }
+
+
+    @TypeConverter
+    fun fromActors(actors : List<Actor>) : String {
+        val gson = Gson()
```

Comment:
here also same moment

Reply:
made new commit with separate tables for each entity